src/youtube-player/youtube-player.spec.ts

```diff
@@ -0,0 +1,121 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {Component, ViewChild} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {YouTubePlayerModule} from './index';
+import {YouTubePlayer} from './youtube-player';
+
+const VIDEO_ID = 'a12345';
+
+const playerState = {
+  UNSTARTED: -1,
+  CUED: 5,
+  OTHER: 9,
+};
```

Comment:
Could this be a TS enum?

Reply:
The problem is the assignment to expose this constant on the window, and, AFAIK, there isn't a way to coerce typescript into allowing an enum to be assigned to a field.